library/include/modules/Job.h

```diff
@@ -102,7 +102,7 @@ namespace DFHack
                                          int filter_idx = -1, int insert_idx = -1);
 
         DFHACK_EXPORT bool isSuitableItem(df::job_item *item, df::item_type itype, int isubtype);
-        DFHACK_EXPORT bool isSuitableMaterial(df::job_item *item, int mat_type, int mat_index);
+        DFHACK_EXPORT bool isSuitableMaterial(df::job_item *item, int mat_type, int mat_index, df::item_type itype = df::item_type::NONE);
```

Comment:
`job_item` objects have an `item_type` field - would that work, or is it different / not set in the case you care about?